docs/userguide/dockervolumes.md

```diff
@@ -216,7 +216,8 @@ then mounting the volumes from the `dbstore` container hides the
 from the `dbstore` container are visible.
 
 You can use multiple `--volumes-from` parameters to bring together multiple data
-volumes from multiple containers.
+volumes from multiple containers. More details about `--volumes-from` can be check
```

Comment:
@HackToday no, I meant just the text she provided, it's slightly different than the text that's currently there;

```Markdown
You can use multiple `--volumes-from` parameters to combine data volumes from
several containers. To find detailed information about `--volumes-from` see the
[mount volumes example](../reference/commandline/run.md#mount-volumes-from-container-volumes-from) in the `run` command reference.
```